packages/client/components/MeetingCard.tsx

```diff
@@ -74,7 +74,8 @@ const Meta = styled('span')({
 const BACKGROUND_COLORS = {
   retrospective: PALETTE.GRAPE_500,
   action: PALETTE.AQUA_400,
-  poker: PALETTE.TOMATO_400
+  poker: PALETTE.TOMATO_400,
+  teamPrompt: `#91E8B7` // TODO: there's no such color in our palette
```

Comment:
I’ll take a look. There may be a variant, but we only added variants from the palette to the codebase as we used them. We didn’t add all of them. I thought it would be a good way to keep track of what we’re using and not. Good eagle eye!

Reply:
Turns out this is the value for `JADE_300` from [our palette](https://www.figma.com/file/OA9NkpSTlHVqqRL9IE9KsF/Palette-v3?node-id=15%3A2184) — could you add the constant to our palette?

Reply:
perfect, just added it. thank you!